src/VerifyAccountHandler.ts

```diff
@@ -23,7 +23,8 @@ export const logger = new Logger({
 
 const metrics = new Metrics({ namespace: POWERTOOLS_METRICS_NAMESPACE, serviceName: POWERTOOLS_SERVICE_NAME });
 
-let HMRC_TOKEN: string;
+let VENDOR_TOKEN: string;
```

Comment:
Can these be declared inside the handler I don't see them being used anywhere else